neqo-qpack/src/decoder.rs

```diff
@@ -553,7 +554,8 @@ impl QPackDecoder {
             if reader.done() {
                 // Send header_ack
                 if req_inserts != 0 {
-                    self.header_ack(stream_id);
+                    let acknowledged_inserts_count = req_inserts - base;
+                    self.header_ack(stream_id, acknowledged_inserts_count);
```

Comment:
It would be good if you could also check that the highest index mentioned in an instruction matches the header~/code/interdependency-trust.

Reply:
Confirmed with @martinthomson that this is about making sure that a header that says it needs access to the Nth entry in a table actually references that entry. 

@martinthomson is going to investigate whether or not this is required by the spec or not.